cvat-ui/src/components/annotation-page/styles.scss

```diff
@@ -128,6 +128,10 @@ button.cvat-predictor-button {
     pointer-events: none;
 }
 
+.ant-btn.cvat-button-active {
+    color: #40a9ff;
```

Comment:
Please define color constant in ``base.scss``, or can we use something already defined?

Reply:
Yes, there is constant for this color already. So we can use it